src/components/UniversalTable/UniversalTable.tsx

```diff
@@ -0,0 +1,30 @@
+import {  ReactNode } from 'react';
+import classes from './UniversalTable.module.scss';
+
+type UniversalTableProps<D> = {
+  data: D[]
```

Comment:
Please create a separate types file in this folder and put this type there

Reply:
Done